src/filter.rs

```diff
@@ -7,6 +7,9 @@ pub fn is_covered(result: &CovResult) -> bool {
         .lines
         .values()
         .any(|&execution_count| execution_count != 0);
+    if any_line_covered == false {
```

Comment:
Since we are checking this earlier now, we shouldn't check it again at line 20.
Instead of `any_line_covered == false`, use `!any_line_covered`.